django_redshift_backend/base.py

```diff
@@ -106,6 +120,12 @@ def alter_index_together(self, model, old_index_together, new_index_together):
     def _create_index_sql(self, model, fields, suffix="", sql=None):
         raise NotImplementedError("Redshift doesn't support INDEX")
 
+    def add_index(self, model, index):
+        pass
+
+    def remove_index(self, model, index):
+        pass
```

Comment:
I think `add_index` and `remove_index` since django-1.11 are not a part of supporting `UUIDField`. 
Is it correct? and what happen without these lines?

refs:
https://github.com/django/django/commit/156e2d59cf92eb252c2f6ef9bb0b65f26c707de2
https://github.com/django/django/commit/52442898e747e9d76bf6349938fa3dada6f0e887

Reply:
Sorry, been a while since I was looking at this, but I believe it had nothing to do with `UUIDField` but everything to do with a model's metaclass defining `index_together`.

Reply:
As this is not directly related to supporting `UUIDField` I can remove this. Let me know.

Reply:
I am not digging about `index_together`, but if this fix is necessary in Django 1.11, regardless of UUIDField, I would like to write it as a change in CHANGES. I understood that this pull request contains two objectives. It is UUIDField support and fixes a bug that migration does not work with Django 1.11. Is this correct?

Reply:
If you remove these two methods, I can merge PR immediately. Otherwise, I need to investigate more about these two methods; checking the effects of versions other than Django-1.11 and what problems it has in 1.11. If you have time, I'm very happy to make another pull request with such information ;)

Reply:
I'll remove the methods. I've actually abandoned Redshift since I created the PR, so investigating this would mean some hassle on my part :)

Reply:
Ah, I understand. I will investigate it separately.